cfn-resources/stream-connection/test/cfn-test-create-inputs.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+# cfn-test-create-inputs.sh
+#
+# This tool generates json files in the inputs/ for `cfn test`.
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+rm -rf inputs
+mkdir inputs
+
+#set profile
+profile="default"
+if [ ${MONGODB_ATLAS_PROFILE+x} ]; then
+	echo "profile set to ${MONGODB_ATLAS_PROFILE}"
+	profile=${MONGODB_ATLAS_PROFILE}
+fi
+
+projectName="${1:-$PROJECT_NAME}"
+echo "$projectName"
+projectId=$(atlas projects list --output json | jq --arg NAME "${projectName}" -r '.results[] | select(.name==$NAME) | .id')
+if [ -z "$projectId" ]; then
+	projectId=$(atlas projects create "${projectName}" --output=json | jq -r '.id')
+
+	echo -e "Created project \"${projectName}\" with id: ${projectId}\n"
+else
+	echo -e "FOUND project \"${projectName}\" with id: ${projectId}\n"
+fi
+echo -e "=====\nrun this command to clean up\n=====\nmongocli iam projects delete ${projectId} --force\n====="
```

Comment:
should we run this command in cfn-test-delete-inputs.sh?

Reply:
we already delete the project in cfn-test-delete-inputs.sh